### ISSUE TITLE
Error on patch, after use shorebird release android --artifact=apk

### ISSUE THREAD

**Issue by JgomesAT** [shorebirdtech/shorebird#857]
I've created a release version with this command shorebird release android --artifact=apk, and all OK

after that, I want to do a patch, but I have an error: I can't see that is try to do and appbundle and not and APK??

```
shorebird patch android -v            
[Process.run] /Users/jorgegomes/.shorebird/bin/cache/flutter/bin/flutter build appbundle --release
✓ Building patch (105.6s)
[HTTP] GET https://api.shorebird.dev/api/v1/apps
✓ Fetching apps (0.3s)
[Process.run] java -jar /Users/jorgegomes/.shorebird/bin/cache/artifacts/bundletool/bundletool.jar dump manifest --bundle=./build/app/outputs/bundle/release/app-release.aab --xpath /manifest/@android:versionName
[Process.run] java -jar /Users/jorgegomes/.shorebird/bin/cache/artifacts/bundletool/bundletool.jar dump manifest --bundle=./build/app/outputs/bundle/release/app-release.aab --xpath /manifest/@android:versionCode
✓ Detected release version 3.0.0+200 (0.4s)
[HTTP] GET https://api.shorebird.dev/api/v1/releases?appId=799ee1a3-d838-4032-8954-4b06364c6be8
✓ Fetching releases (0.2s)
[Process.run] git rev-parse HEAD (in /Users/jorgegomes/.shorebird/bin/cache/flutter)
✓ Fetching Flutter revision (10ms)
[HTTP] GET https://api.shorebird.dev/api/v1/releases/2060/artifacts?arch=aarch64&platform=android
⠹ Fetching release artifacts... (0.2s)[HTTP] GET https://api.shorebird.dev/api/v1/releases/2060/artifacts?arch=arm&platform=android
⠼ Fetching release artifacts... (0.3s)[HTTP] GET https://api.shorebird.dev/api/v1/releases/2060/artifacts?arch=x86_64&platform=android
✓ Fetching release artifacts (0.5s)
[HTTP] GET https://api.shorebird.dev/api/v1/releases/2060/artifacts?arch=aab&platform=android
✓ Fetching aab artifact (0.2s)
✓ Downloading release artifacts (5.7s)
Unhandled exception:
Bad state: No element
#0      ListBase.firstWhere (dart:collection/list.dart:132:5)
#1      AabDiffer._metaInfMfContent (package:shorebird_cli/src/archive_analysis/aab_differ.dart:50:12)
#2      AabDiffer.changedFiles (package:shorebird_cli/src/archive_analysis/aab_differ.dart:34:27)
#3      PatchAndroidCommand.run (package:shorebird_cli/src/commands/patch/patch_android_command.dart:240:37)
<asynchronous suspension>
#4      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#5      ShorebirdCliCommandRunner.runCommand (package:shorebird_cli/src/command_runner.dart:151:18)
<asynchronous suspension>
#6      ShorebirdCliCommandRunner.run (package:shorebird_cli/src/command_runner.dart:93:14)
<asynchronous suspension>
#7      main (file:///Users/jorgegomes/.shorebird/packages/shorebird_cli/bin/shorebird.dart:19:5)
<asynchronous suspension>
```

**Comment by eseidel** [comment]
Looks like it's failing because one of the aab's is empty?
https://github.com/shorebirdtech/shorebird/blob/main/packages/shorebird_cli/lib/src/archive_analysis/aab_differ.dart#L45

**Comment by eseidel** [comment]
Thanks for the report!

FYI @bryanoltman

**Comment by JgomesAT** [comment]
HI,

And, do you know how I can solve this? in my project don't use aab file

**Comment by eseidel** [comment]
I don't have a solution for you yet, but we can look.  Thank you for the report!

**Comment by JgomesAT** [comment]
The problem is not solved in version 0.11 and ticket is closed

Fetching release artifacts... (0.3s)[HTTP] GET https://api.shorebird.dev/api/v1/apps/b6a80ead-ae84-4729-87d3-b3d30300b869/releases/2802/artifacts?arch=x86_64&platform=android
✓ Fetching release artifacts (0.5s)
[HTTP] GET https://api.shorebird.dev/api/v1/apps/b6a80ead-ae84-4729-87d3-b3d30300b869/releases/2802/artifacts?arch=aab&platform=android
✓ Fetching aab artifact (0.2s)
✓ Downloading release artifacts (5.5s)
Unhandled exception:
Bad state: No element
#0      ListBase.firstWhere (dart:collection/list.dart:132:5)
#1      AabDiffer._metaInfMfContent (package:shorebird_cli/src/archive_analysis/aab_differ.dart:50:12)
#2      AabDiffer.changedFiles (package:shorebird_cli/src/archive_analysis/aab_differ.dart:34:27)
#3      PatchAndroidCommand.run (package:shorebird_cli/src/commands/patch/patch_android_command.dart:245:37)
<asynchronous suspension>
#4      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#5      ShorebirdCliCommandRunner.runCommand (package:shorebird_cli/src/command_runner.dart:151:18)
<asynchronous suspension>
#6      ShorebirdCliCommandRunner.run (package:shorebird_cli/src/command_runner.dart:93:14)
<asynchronous suspension>
#7      main (file:///Users/jorgegomes/.shorebird/packages/shorebird_cli/bin/shorebird.dart:22:5)
<asynchronous suspension>

**Comment by felangel** [comment]
This should be fixed on main by https://github.com/shorebirdtech/shorebird/pull/922 and will be available in the next stable release.
